.github/ISSUE_TEMPLATE/something-is-incorrect.md

```diff
@@ -0,0 +1,67 @@
+---
+name: Something is incorrect
+about: A code sample, signature or definition, heading or title, some prose, or formatting
+  is wrong
+title: ''
+labels: docs
+assignees: ''
+
+---
+
+<!--
+  Feel free to delete anything that definitely does not apply to your
+  issue, as well as these comments. (Or don’t, it’s up to you.)
+
+  FIRST!: Check the site at https://perl6docs.github.io/ rather than
```

Comment:
OK, I guess that's something new?

Reply:
It’s referenced in several places, such as:

https://github.com/perl6/doc/blob/58b9ef05015db6b58d7300e967eb31fb9020c0f6/README.md#L5-L9